src/main/java/dk/cngroup/lentils/service/CypherService.java

```diff
@@ -51,4 +62,16 @@ public void saveAll(List<Cypher> cyphers) {
     public List<Cypher> getAll() {
         return cypherRepository.findAll();
     }
+
+    public int getScore(Cypher cypher, Team team) {
+        int statusScore = statusService.getStatusScore(team, cypher );
+        int hintScore = hintTakenService.getHintScore(team, cypher);
+        return statusScore - hintScore;
+    }
+
+    public int getScore(Long cypherId, Long teamId) {
+        Optional<Team> team = teamService.get(teamId);
```

Comment:
Tady uz bych to Optional nepouzival. Je to michani error handlingu a bussiness logiky ta service uz by mela vratit team a nebo vyhodit vyjimka - v praci vysvetlim ;)